lib/authorizer/oauth2.js

```diff
@@ -22,7 +29,7 @@ module.exports = {
      * @param {AuthHandlerInterface~authPreHookCallback} done
      */
     pre: function (auth, done) {
-        done(null, true);
+        done(null, Boolean(auth.get('accessToken')));
```

Comment:
Consider implementing `auth.has` to avoid repetitive `Boolean` casts.

Reply:
the check is not universal and can depend on the Auth and the parameter itself